<launch>

  <!-- Nodelet manager for this pipeline -->
  <node pkg="nodelet" type="nodelet" args="manager"
        name="record_player_manager" output="screen"/>

  <!-- Convert it into a point cloud -->
  <node pkg="nodelet" type="nodelet" name="cloudify"
        args="load depth_image_proc/point_cloud_xyzrgb record_player_manager --no-bond">
    <remap from="depth_registered/image_rect" to="kinect1/depth_registered/filtered"/>
    <remap from="depth_registered/points" to="kinect1/depth_registered/filtered/points"/>
    <remap from="rgb/image_rect_color" to="kinect1/rgb/image_rect_color"/>
    <remap from="rgb/camera_info" to="/kinect1/rgb/camera_info"/>
  </node>
  
</launch>